packages/store/package.json

```
@@ -14,7 +14,9 @@
     "lint-fix": "yarn lint:eslint --fix",
     "lint": "run-s --continue-on-error lint:*",
     "lint:types": "tsc",
-    "lint:eslint": "eslint ."
+    "lint:eslint": "eslint .",
+    "prepack": "tsc --build tsconfig.build.json",
+    "postpack": "git clean -f ':!exported.d.ts' '*.d.ts*' '*.tsbuildinfo'"
```

please make this consistent as well:
```suggestion
    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
```

---

Ah, of course!

---

Ah, of course! Ready https://github.com/Agoric/agoric-sdk/pull/10917/commits/5d590aae911ad5c353b5d7066f9d35d83bd7f179 I've made everything one commit.